app/src/controllers/testItem/selectors.js

```diff
@@ -300,47 +300,62 @@ export const statisticsLinkSelector = createSelector(
   debugModeSelector,
   testItemIdsArraySelector,
   (query, payload, testItemIds, isDebugMode, testItemIdsArray) => (ownProps) => {
-    const linkPayload = (ownProps.ownLinkParams && ownProps.ownLinkParams.payload) || payload;
-    const launchesLimit = ownProps.launchesLimit;
-    const providerType = ownProps.providerType;
+    const {
+      launchesLimit,
+      providerType,
+      isLatest,
+      ownLinkParams,
+      itemId,
+      keepFilterParams,
+      attributeKey,
+      attributeValue,
+      compositeAttribute,
+      levelAttribute,
+      baselineLaunchId,
+      launchId,
+      statuses,
+      startTime,
+      types,
+      excludeSkipped,
+    } = ownProps;
+    const linkPayload = (ownLinkParams && ownLinkParams.payload) || payload;
     const providerTypeModifierId = PROVIDER_TYPE_MODIFIERS_ID_MAP[providerType];
-    const isLatest = ownProps.isLatest;
-    const page =
-      (ownProps.ownLinkParams && ownProps.ownLinkParams.page) || getNextPage(isDebugMode, true);
+    const page = (ownLinkParams && ownLinkParams.page) || getNextPage(isDebugMode, true);
     let levelIndex = 0;
     if (testItemIdsArray.length > 0) {
-      levelIndex = !ownProps.itemId ? testItemIdsArray.length - 1 : testItemIdsArray.length;
+      levelIndex = !itemId ? testItemIdsArray.length - 1 : testItemIdsArray.length;
     }
     const queryNamespace = getQueryNamespace(levelIndex);
     const params = {
-      ...(ownProps.keepFilterParams ? extractNamespacedQuery(query, queryNamespace) : {}),
+      ...(keepFilterParams ? extractNamespacedQuery(query, queryNamespace) : {}),
       'filter.eq.hasStats': true,
       'filter.eq.hasChildren': false,
       'filter.in.type': LEVEL_STEP,
-      'filter.has.attributeKey': ownProps.attributeKey,
-      'filter.has.attributeValue': ownProps.attributeValue,
-      'filter.has.compositeAttribute': ownProps.compositeAttribute,
-      'filter.has.levelAttribute': ownProps.levelAttribute,
+      'filter.has.attributeKey': attributeKey,
+      'filter.has.attributeValue': attributeValue,
+      'filter.has.compositeAttribute': compositeAttribute,
+      'filter.has.levelAttribute': levelAttribute,
       providerType,
       [providerTypeModifierId]: ownProps[providerTypeModifierId],
       launchesLimit,
       isLatest,
-      ...(ownProps.baselineLaunchId && { baselineLaunchId: ownProps.baselineLaunchId }),
-      ...(ownProps.launchId && { launchId: ownProps.launchId }),
+      ...(baselineLaunchId && { baselineLaunchId }),
+      ...(launchId && { launchId }),
+      excludeSkipped,
```

Comment:
I suppose to meet the ticket requirements it is enough to just not provide 'skipped' param while navigating from the widget.
The filters on test item level shouldn't be changed as they are just reflect to the query provided. If it doesn't contain 'skipped', it won't be selected in refines, but should be visible for user to switch it on/off as all other filters.